Merge rel/latest to master

byhow opened the pull request:
Back merge release. This includes all modules.

Ticket: BG-68445

byhow committed:
chore(release): publish modules

 - @bitgo/abstract-eth@1.2.14
 - @bitgo/abstract-utxo@2.0.0
 - @bitgo/account-lib@4.4.0
 - bitgo@18.0.0
 - @bitgo/blockapis@1.6.0
 - @bitgo/express@9.24.0
 - @bitgo/key-card@0.1.0
 - @bitgo/sdk-api@1.9.9
 - @bitgo/sdk-coin-ada@2.3.9
 - @bitgo/sdk-coin-algo@1.3.14
 - @bitgo/sdk-coin-atom@2.0.0
 - @bitgo/sdk-coin-avaxc@2.4.5
 - @bitgo/sdk-coin-avaxp@3.5.5
 - @bitgo/sdk-coin-bch@1.3.14
 - @bitgo/sdk-coin-bcha@1.5.9
 - @bitgo/sdk-coin-bsc@3.3.0
 - @bitgo/sdk-coin-bsv@1.3.14
 - @bitgo/sdk-coin-btc@1.3.5
 - @bitgo/sdk-coin-btg@1.3.14
 - @bitgo/sdk-coin-celo@1.3.14
 - @bitgo/sdk-coin-cspr@1.2.14
 - @bitgo/sdk-coin-dash@1.3.14
 - @bitgo/sdk-coin-doge@1.8.9
 - @bitgo/sdk-coin-dot@2.2.3
 - @bitgo/sdk-coin-eos@1.3.14
 - @bitgo/sdk-coin-etc@1.2.14
 - @bitgo/sdk-coin-eth@4.2.0
 - @bitgo/sdk-coin-eth2@1.2.14
 - @bitgo/sdk-coin-ethw@1.3.14
 - @bitgo/sdk-coin-hbar@1.3.14
 - @bitgo/sdk-coin-ltc@2.0.9
 - @bitgo/sdk-coin-near@1.3.14
 - @bitgo/sdk-coin-polygon@1.4.7
 - @bitgo/sdk-coin-rbtc@1.3.14
 - @bitgo/sdk-coin-sol@2.3.8
 - @bitgo/sdk-coin-stx@1.4.14
 - @bitgo/sdk-coin-sui@2.0.0
 - @bitgo/sdk-coin-trx@1.6.3
 - @bitgo/sdk-coin-xlm@2.0.0
 - @bitgo/sdk-coin-xrp@1.3.14
 - @bitgo/sdk-coin-xtz@1.4.14
 - @bitgo/sdk-coin-zec@1.3.14
 - @bitgo/sdk-core@7.0.0
 - @bitgo/sdk-test@1.2.14
 - @bitgo/sdk-unified-wallet@2.0.0
 - @bitgo/statics@14.3.0
 - @bitgo/unspents@0.11.12
 - @bitgo/utxo-bin@2.7.3
 - @bitgo/utxo-lib@7.7.0
 - @bitgo/web-demo@1.5.0

byhow committed:
build(express): update dockerfile to latest

Ticket: BG-68445

LeoKotschenreuther reviewed (commented):
Automatic comment to flush this PR from my review queue. 🚽🚽🚽

byhow committed:
Merge branch 'master' into rel/latest

byhow committed:
chore(release): publish modules

 - @bitgo/abstract-eth@1.2.15
 - @bitgo/abstract-utxo@2.0.1
 - @bitgo/account-lib@4.4.1
 - bitgo@18.0.1
 - @bitgo/express@9.24.1
 - @bitgo/key-card@0.1.1
 - @bitgo/sdk-api@1.9.10
 - @bitgo/sdk-coin-ada@2.3.10
 - @bitgo/sdk-coin-algo@1.3.15
 - @bitgo/sdk-coin-atom@2.1.0
 - @bitgo/sdk-coin-avaxc@2.4.6
 - @bitgo/sdk-coin-avaxp@3.5.6
 - @bitgo/sdk-coin-bch@1.3.15
 - @bitgo/sdk-coin-bcha@1.5.10
 - @bitgo/sdk-coin-bsc@3.3.1
 - @bitgo/sdk-coin-bsv@1.3.15
 - @bitgo/sdk-coin-btc@1.3.6
 - @bitgo/sdk-coin-btg@1.3.15
 - @bitgo/sdk-coin-celo@1.3.15
 - @bitgo/sdk-coin-cspr@1.2.15
 - @bitgo/sdk-coin-dash@1.3.15
 - @bitgo/sdk-coin-doge@1.8.10
 - @bitgo/sdk-coin-dot@2.2.4
 - @bitgo/sdk-coin-eos@1.3.15
 - @bitgo/sdk-coin-etc@1.2.15
 - @bitgo/sdk-coin-eth@4.2.1
 - @bitgo/sdk-coin-eth2@1.2.15
 - @bitgo/sdk-coin-ethw@1.3.15
 - @bitgo/sdk-coin-hbar@1.3.15
 - @bitgo/sdk-coin-ltc@2.0.10
 - @bitgo/sdk-coin-near@1.3.15
 - @bitgo/sdk-coin-polygon@1.4.8
 - @bitgo/sdk-coin-rbtc@1.3.15
 - @bitgo/sdk-coin-sol@2.3.9
 - @bitgo/sdk-coin-stx@1.4.15
 - @bitgo/sdk-coin-sui@2.0.1
 - @bitgo/sdk-coin-trx@1.6.4
 - @bitgo/sdk-coin-xlm@2.0.1
 - @bitgo/sdk-coin-xrp@1.3.15
 - @bitgo/sdk-coin-xtz@1.4.15
 - @bitgo/sdk-coin-zec@1.3.15
 - @bitgo/sdk-core@7.0.1
 - @bitgo/sdk-test@1.2.15
 - @bitgo/sdk-unified-wallet@2.0.1
 - @bitgo/statics@14.4.0
 - @bitgo/utxo-bin@2.7.4
 - @bitgo/web-demo@1.5.1

byhow committed:
build(express): update dockerfile to latest

Ticket: BG-68527

OttoAllmendinger reviewed (commented):
Automatic comment to flush this PR from my review queue. 🚽🚽🚽

jeanlarrieux commented:
Looks good for `sdk-unified-wallet` (`defi-team`)
